.mailmap

```diff
@@ -583,3 +583,5 @@ Rahil Parikh <r.parikh@somaiya.edu> rprkh <r.parikh@somaiya.edu>
 Yaser AlOsh <yaseralosh@outlook.com> Yaser <yaseralosh@outlook.com>
 Joannah Nanjekye <joannah.nanjekye@ibm.com> nanjekyejoannah <joannah.nanjekye@ibm.com>
 Kshitij <kshitijparwani.mat18@itbhu.ac.in> Kshitij Parwani <44468674+P-Kshitij@users.noreply.github.com>
+Moha Jain <mohajain99@gmail.com> mohajain <mohajain99@gmail.com>
+Moha Jain <mohajain99@gmail.com> mohajain <45903778+mohajain@users.noreply.github.com>
```

Comment:
This doesn't seem right.

Reply:
I simply followed the steps mentioned above. What's wrong with this?

Reply:
@sidhu1012 Please tell me if there is some problem with this. I was added in AUTHORS when oscarbenjamin was updating the authors and mailmap and he added my entries in the same way. I have also read through the instructions in mailmap and authors file and I believe this is correct although I may be wrong. Maybe oscarbenjamin can confirm if I am wrong.

Reply:
This looks fine to me